src/scripts/generate-test-patient-spreadsheets.sh

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env bash
+set -euo pipefail
+if [ "${DEBUG:=false}" == "true" ]; then set -x; fi
+export WORK=$(mktemp -p /tmp -d work.XXXX)
+
+main() {
+  local outputXlsx="${2}" tmpOutputCsv
+  TEMPLATE_FILE="${1}"
```

Comment:
I would do all or nothing for these. Either process them both as local vars or store them both as globals. I Think the either or could get confusing for anyone making updates to the script.